src/resources/Catalogs/Catalog.ts

```diff
@@ -32,6 +32,9 @@ export default class Catalog extends Resource {
         return this.api.get<CatalogModel>(`${Catalog.baseUrl}/${catalogId}`);
     }
 
+    /**
+     * @deprecated `you should now use `getFieldStats(catalogId: string, options?: CatalogFieldStatsOptions)`.
```

Comment:
Should we indicate that the method is left for backwards-compatibility like we do for `update` a little below? :eyes:

Reply:
Good idea! Change made. Thanks.

Reply:
I still see it as it was previously :eyes: